### PR TITLE
Add SonataAnnotationBundle to docs

### PR DESCRIPTION
**Pull request by kunicmarko20**
I am targeting this branch, because this is docs.

## Subject

Adding [SonataAnnotationBundle](https://github.com/kunicmarko20/SonataAnnotationBundle) to Sonata docs.

### CODE REVIEW

File: docs/cookbook/recipe_creating_an_admin_with_annotations.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+You are done and you probably want to know how this looks like in the admin
+interface. Well, let's find out by going to http://localhost:8000/admin
+
+.. image:: ../images/admin_with_annotations.png
+   :align: center
+   :alt: Sonata Dashboard with SonataAnnotationBundle
+   :width: 700px
+
+Rest of the Annotation and their options can be found `here`_.
```
Comment (greg0ire) [inline docs/cookbook/recipe_creating_an_admin_with_annotations.rst]: "The rest"

**Comment by kunicmarko20** [comment]
Fixed, it looks like I am getting better with writing :smile:

**Comment by greg0ire** [comment]
It does :)

**Comment by OskarStark** [comment]
Thank you @kunicmarko20 👍

**Pull request merged**